src/sentry/models/minidump.py

```diff
@@ -8,13 +8,21 @@
 
 from __future__ import absolute_import
 
+import re
+import six
 from django.db import models, transaction
 from symbolic import ProcessState
 
 from sentry.constants import LOG_LEVELS_MAP
 from sentry.db.models import FlexibleForeignKey, Model, sane_repr
 from sentry.models.file import File
 
+_version_re = re.compile(r'(\d+\.\d+\.\d+)\s+(.*)')
+
+_minidump_os_mapping = {
+    'Mac OS X': 'macOS',
```

Comment:
Breakpad uses a different naming schema for operating systems than what we have hardcoded in the symbolizer. For the sake of keeping this change small, I'm just mapping the names here. However, we should rather try not to use the os name in the pipeline. UI is a separate topic...